src/actions.cpp

```diff
@@ -747,4 +766,12 @@ void evm_contract::assertnonce(eosio::name account, uint64_t next_nonce) {
     }
 }
 
+void evm_contract::setversion(uint64_t version) {
+    require_auth(get_self());
+    auto config = _config.get();
+    eosio::check(version <= MAX_EOSEVM_SUPPORTED_VERSION, "Unsupported version");
```

Comment:
The check that the `version` is greater than the current one is done within the `set_version` call. So why do this check that the version isn't too large here rather than in the same place as the `set_version` call?